src/NuGet.Indexing/ResponseFormatter.cs

```diff
@@ -346,7 +346,19 @@ public static void WriteStatsResult(JsonWriter jsonWriter, NuGetIndexSearcher se
             jsonWriter.WriteStartObject();
             WriteProperty(jsonWriter, "numDocs", searcher.IndexReader.NumDocs());
```

Comment:
No `WriteProperty<T>` is our code not some magic that c# does. It will not handle it we need to add the logic to identify and serialize object. http://www.newtonsoft.com/json/help/html/M_Newtonsoft_Json_JsonWriter_WriteValue_29.htm